packages/lab/src/dialog/DialogHeader.css

```diff
@@ -0,0 +1,63 @@
+/* Styles applied to the root element */
+.saltDialogHeader {
+  padding-bottom: var(--salt-spacing-300);
+  padding-left: var(--salt-spacing-300);
+  padding-right: var(--salt-spacing-300);
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: var(--salt-spacing-100);
+  box-sizing: border-box;
+}
+
+.saltDialogHeader-container {
+  flex-grow: 1;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  gap: var(--salt-spacing-50);
+}
+
+.saltDialogHeader-header {
+  margin: 0;
+}
+
+.saltDialogHeader-actionsContainer {
+  align-self: flex-start;
+}
+
+/* Styles applied to the status indicator icon overriding its default size */
+.saltDialogHeader .saltStatusIndicator.saltIcon {
+  --icon-size: var(--salt-text-h2-lineHeight);
+}
+
+/* Styles applied to DialogHeader when accent={true} */
+.saltDialogHeader-withAccent {
+  position: relative;
+}
+
+.saltDialogHeader-withAccent::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: var(--salt-spacing-300);
+  width: var(--salt-size-bar);
+  background: var(--salt-accent-background);
+}
+
+.saltDialogHeader-error::before {
```

Comment:
Not too sure what `::before` is setting, i don't see the accent bar when validation status is set?